Sources/AppcuesKit/Presentation/Debugger/ScreenCapturing/UIKitElementTargeting.swift

```diff
@@ -16,21 +16,30 @@ internal class UIKitElementSelector: AppcuesElementSelector {
         case accessibilityIdentifier
         case accessibilityLabel
         case tag
+        case tab
     }
 
     let accessibilityIdentifier: String?
     let tag: String?
     let appcuesID: String?
-
-    init?(appcuesID: String?, accessibilityIdentifier: String?, accessibilityLabel: String?, tag: String?) {
+    let tab: String?
```

Comment:
This is more ramblings than a concrete suggestion, but...

I don't know if there'll be other UIKit things we'll one day want to automatically add selectors for, but it could get unwieldy quickly if we add a new named value like `tab` every time. We could maybe do a more generic name here (idk what) and we could reuse across mutually exclusive UIKit types. Like if wanted to automatically tag UIBarButtonItem's or UISwitch's or something...

Reply:
Yeah, good call. Pondering names.

viewData, viewInfo, descriptor, generated, autoTag...

Reply:
I like `autoTag`, maybe because it's in the same vein as our auto-properties? Maybe `autoSelector`? (sounds like car shopping... 🙃)

Reply:
Ok lets roll with `autoTag` then for now, seems decent enough.